src/skadi/sample.cc

```diff
@@ -353,6 +390,7 @@ tile_data cache_t::source(uint16_t index) {
       }
     }
   }
+
   if (!unpacked) {
     unpacked = (char*)malloc(HGT_BYTES);
```

Comment:
^^ this is why we can use `const_cast`, because here `unpacked` is not actually `const` so casting it away will not be undefined behavior